### DISCUSSION TITLE
mirai console插件怎么编译

### DISCUSSION THREAD

**Discussion by feassh** [mamoe/mirai-console#259] [问答] [answered]
mirai console插件怎么编译啊，开发好了，直接编译成jar不能加载，github没有说编译要准备哪些东西。。。。
有大佬知道吗

**Comment by Him188** [comment, marked as answer]
执行 Gradle task `buildPlugin`. 请先确保 service 配置正常.

**Reply by Him188** [reply]
https://github.com/mamoe/mirai-console/blob/master/docs/Plugins.md#%E7%A1%AE%E8%AE%A4%E4%B8%BB%E7%B1%BB%E6%AD%A3%E7%A1%AE%E5%AE%9A%E4%B9%89

**Reply by feassh** [reply]
可以了  感谢！！
